tools/release-note.md

```diff
@@ -1,7 +1,7 @@
 This is release `${RELEASE_TAG}` of the Grafana Agent.
 
 ### Upgrading
-Read the [upgrade guide](https://github.com/grafana/agent/blob/${RELEASE_TAG}/docs/upgrade-guide/_index.md) for specific instructions on upgrading from older versions.
+Read the [upgrade guide](https://github.com/grafana/agent/blob/${RELEASE_TAG}/docs/sources/upgrade-guide/_index.md) for specific instructions on upgrading from older versions.
```

Comment:
Can we change this to https://grafana.com/docs/agent/latest/upgrade-guide/?

Reply:
`https://grafana.com/docs/agent/v0.26/upgrade-guide`? We'll need some magic to pull out the major and minor version, but I think we should point people at the rendered documentation.

Reply:
Added code to pull out the major/minor version for release docs